Entities/TerrainDebris.cpp

```diff
@@ -58,6 +58,7 @@ namespace RTE {
 			reader >> m_DebrisFile;
 		} else if (propName == "DebrisPieceCount") {
 			reader >> m_BitmapCount;
+			m_Bitmaps.reserve(m_BitmapCount);
```

Comment:
This may actually be pointless unless you change GetAsAnimation to take in the vector as an out param. Though maybe I'm wrong and cpp is smort about this, so even though you're copying shit over the reservation still helps.

Reply:
This would be happening on every create so not constructing a new vector and then reallocating the old one would be nice picogainzz.
I'll keep the current copy return as an overload though.

Reply:
I don't think there's much reason to have the less gainz one. It's just code clutter for no functional difference, and having both means the easier one to use (which is the worse one) will be more likely to be used randomly.

Point being, just replace it.

Also yeah, I was gonna use mobile github to reply to this cause I was away, but didn't wanna bother lol.